CMakeLists.txt

```diff
@@ -155,6 +158,3 @@ if (TESTS)
     enable_testing()
     add_subdirectory(test)
 endif()
-
-# TODO: Split out json_spirit, libscrypt and sec256k1.
-add_subdirectory(utils)
```

Comment:
So its not even needed for `json_spirit`?

Reply:
We use json_spirit by loading headers only. This is an old CMake script that was not used.